docs/usage/usage.md

```diff
@@ -78,6 +78,8 @@ typha:
   enabled: true
 overlay:
   enabled: true
+autoScaling:
+  mode: "vpa"
```

Comment:
Perhaps you could include a brief paragraph detailing the available modes and the conditions that would make one mode preferable over another. If I understand correctly, the VPA mode will only function if VPA is enabled for the shoot. This could be worth mentioning.